src/clustering/administration/issues/non_transitive.cc

```diff
@@ -73,5 +73,5 @@ std::vector<scoped_ptr_t<issue_t> > non_transitive_issue_tracker_t::get_issues(
     if (interruptor->is_pulsed()) {
         throw interrupted_exc_t();
     }
-    return std::move(issues);
```

Comment:
Definitely not. I just run `run-clang-tidy.py src -checks=-*,modernize-use-nullptr -fix`